### ISSUE TITLE
Feature request : remove duplicates from source folder

### ISSUE THREAD

**Issue by kifeo** [andrewning/sortphotos#129]
It would be very nice to be able to remove the duplicates from the source folder, so that there is no remaining processed image in the source folder

**Comment by TRSx80** [comment]
Yes, it looks like "[duplicate removal](https://github.com/andrewning/sortphotos#duplicate-removal)" does not actually do what you and I would expect, which (IMO) is more clearly defined as the title of this issue.

However, after examining my logs, it seems that `sortphotos` does do a very good job of making sure these files already exist in the destination.  It just doesn't delete them from the source.

And therefore, if you are moving (not copying) them, like I am, I guess you could simply delete whatever files are left in your source folder right after `sortphotos` runs.  In my case, I run it nighty, so I may just add something like `rm /source/folder/*` to run right after `sortphotos`.

If you want to be paranoid, you could go through one by one and make sure the duplicate source files do in fact exist in the destination.  I am, so I just finished doing this.  But now that I see how `sortphotos` works, I may do the above.  Or just manually go through and compare them periodically (I haven't decided yet).  FWIW, I seem to have 35 files that were not moved / deleted for whatever reason[0], and this in a span of time going back to 2021-06-29 (the date of first file, about 7 months).

---

Andrew, if you read this, I just wanted to thank you for sharing this tool.  It has certainly helped me (and apparently, a lot of other people!).  In fact it looks like it has become popular to the point of becoming perhaps a bit of a burden for you, which is unfortunate.  Anyway, cheers!  :beers:

---

[0] In my case, I suspect some weirdness with either Syncthing (which I am using to sync both source and destination folders), or (even more likely) the fact my sd card is getting full on my phone as the original reason these were not deleted in the first place (in other words, not `sortphotos`).
